apparmor.d/groups/full-policy/full-policy

```diff
@@ -0,0 +1,251 @@
+  deny capability ipc_owner,
+  deny capability sys_tty_config,
+  deny capability mac_admin, # intentional to protect policy
+  deny capability mac_override, # intentional to protect policy
+  deny capability sys_module,
+  deny capability linux_immutable,
+  deny capability lease,
+  deny capability net_broadcast,
+  deny capability perfmon,
+  deny capability sys_boot,
+  deny capability sys_pacct,
+  deny capability sys_time,
+  deny capability wake_alarm,
+  deny capability setfcap,
+  
+  ## will break steam without profile
+  deny pivot_root, 
+
+  ## will break k3s without profile
+  deny unix (listen),
+  
+  ## won't break with the current set of profiles
+  deny unix (create),
+  deny unix (getattr),
+  deny unix (setattr),
+  deny unix (setopt),
+  deny unix (getopt),
+  
+  ## won't break with the current set of profiles
+  deny ptrace (trace),
+  deny ptrace (tracedby),
+  deny ptrace (readby),
+  
+  ## won't break with the current set of profiles
+  deny network bluetooth,
+  deny network alg,
+  deny network ash,
+  deny network rose,
+  deny network x25,
+  deny network ax25,
+  deny network ipx,
+  deny network netrom,
+  deny network appletalk,
+  deny network econet,
+  deny network qipcrtr,
+  deny network bridge,
+  deny network atmpvc,
+  deny network netbeui,
+  deny network security,
+  deny network key,
+  deny network atmsvc,
+  deny network rds,
+  deny network irda,
+  deny network pppox,
+  deny network wanpipe,
+  deny network ib,
+  deny network mpls,
+  deny network can,
+  deny network tipc,
+  deny network rxrpc,
+  deny network isdn,
+  deny network phonet,
+  deny network ieee802154,
+  deny network caif,
+  deny network vsock,
+  deny network kcm,
+  deny network smc,
+  deny network xdp,
+  
+  # -----
+  
+  ## Section 2 - File permissions
+  
+  ## This is quite restrictive for a "general" profile.
+  ## Can of course be further restricted. Probably by a lot.
+  
+  ## The owner can read pretty much everything
+  ## He can also write to the directories
+  ## directly under root.
+  / r,
+  owner / rwlk,
+  owner /** rw,
+
+  ## Everyone can see the home directories
+  ## Only the owners allowed inside
+  /home r,
+  owner /home/** rwlkPix,
+
+  ## Reserved for the owner 'root' only
+  owner /boot/** rwlk,
+  owner /root/** rwlk,
+
+  ## Running stuff is fine here
+  ## Modifying requires ownership
+  @{lib}/** rPix,
+  owner @{lib}/** rwmlkPix,
+  # Here too
+  @{bin}/** rPix,
+  owner @{bin}/** rwmlkPix,
+  # And here
+  /opt/** rPix,
+  owner /opt/** rwmlkPix,
+  
+  ## Owner can modify only
+  /usr/** r,
+  owner /usr/** rwlk,
+
+  ## Don't you dare read others' temp files
+  owner /{,var/}tmp/** rw,
+ 
+  ## Reading etc is cool
+  ## No modification allowed if not the owner
+  /{,usr/local/}etc/** r,
+  owner /{,usr/local/}etc/** rwmlk,
+
+  ## Can be restricted? Maybe
+  /dev/** rw,
+
+  ## Owner can access his media
+  owner /media/** rw,
+  
+  ## Or what is mounted
+  owner /mnt/** rw,
```

Comment:
Use `@{MOUNTDIR}` instead of `/media/` and `/mnt/`

Reply:
Sure thing.